modules/imgproc/src/contours.cpp

```diff
@@ -284,10 +284,13 @@ cvStartFindContours( void* _img, CvMemStorage* storage,
                                           scanner->cinfo_storage );
     }
 
+    assert(step >= 0);
+    assert(size.height >= 1);
```

Comment:
Thanks!
Yep, there is still a lot of legacy code.
I hope CV_Assert will become more useful sometimes and will provide more information, like stack traces.